pymdb.py

```diff
@@ -148,20 +140,22 @@ def plot(self):
         print "http://www.imdb.com/title/%s"% self.stuff.find('movie').get("imdbID")
 
     def awards(self):
-        print  self.stuff.find('movie').get("awards")
+        return self.stuff.find('movie').get("awards")
 
     def reviews(self):
         """Prints Rotten Tomatoes Critics Consensus"""
-        print  self.stuff.find('movie').get("tomatoConsensus")
+        print self.stuff.find('movie').get("tomatoConsensus")
         print "For more visit: ", self.stuff.find('movie').get("tomatoeURL")
         print "http://www.imdb.com/title/%s/reviews?ref_=tt_ov_rt"% self.stuff.find('movie').get("imdbID")
 
 
 class MovieId(Movie):
     """Takes IMDb ID as parameter instead of title"""
-    def __init__(self,id):
-        url=urllib.urlopen("http://www.omdbapi.com/?i=%s&y=&plot=short&r=xml"% id)
-        input=url.read()
+
+    def __init__(self, id):
+        url = urllib.urlopen("http://www.omdbapi.com/?i=%s&y=&plot=short&r=xml"% id)
+        input = url.read()

```

Comment:
This warning comes up using PyCharm: `Shadows built-in name 'input'`

This variable name should be changed.